src/components/accordion/_accordion.scss

```diff
@@ -38,8 +39,8 @@
 
   .bx--accordion__arrow {
     transition: all $transition--base $bx--standard-easing;
-    height: .75rem;
-    width: .75rem;
+    height: 1.25rem;
+    width: 1.25rem;
```

Comment:
Change: `padding` was affecting size of accordion arrow icons making them appear smaller than normal.